string name 
uint8 frame_id
uint8 height_id
mrs_msgs/Reference[] points
uint8 terminal_action

